source/blog/_posts/2016-11-30-new-cart-bundle.md

```diff
@@ -0,0 +1,31 @@
+---
+title: New cart bundle concept
+tags:
+- cart
+- refactoring
+
+categories:
+- dev
+
+authors: [dr]
+
+github_link: blog/_posts/2016-11-30-new-cart-bundle.md
+---
+
+Since the last Community Day we get many questions about the new shopping cart:
+- *How far are you with the development*
+- *Which new features come with the new shopping cart?*
+
+On community day, we also announced that we are striving for an open development in the refactoring process in order to get as much feedback as possible and to be able to work closely with other developers and partners. We want to realize this now with a first concept of the new shopping cart. You can see the development process on <a href="https://github.com/shopware/shopware-cart-poc">Github</a>, where we created a new repository which allows the community to create pull requests and issues. The new repository contains a new bundle in `/engine/Shopware/Bundle/CartBundle` which contains a first proof of concept for a new cart process. This first concept can change steadily due to growing requirements and ongoing refactorings. First features which we implemented are the following:
+- Add, delete and change quantity of product line items
+- Add and delete percentages vouchers
+- First concept for a partial delivery
+
+A storefront integration isn't implemented now. All classes are only used inside unit test to proof different processes and calculations before the view layer will be implemented. On the technical side of view, the cart contains already the following concepts/features:
```

Comment:
A storefront integration has yet to be implemented. All classes are used inside the unit test to check different processes and calculations before the view layer can be implemented. From a technical perspective, the cart already contains the following concepts/features: